### ISSUE TITLE
It doesn't save the Torrent card settings permanently

### ISSUE THREAD

**Issue by simonbcn** [WDaan/VueTorrent#1272]
```
Google Chrome 119.0.6045.105 
MacOS 14 
VueTorrent (git latest-release) 
```

I'm not sure what version of VueTorrent I'm actually running because in the settings section it says it's version 2.0.0: ![Screenshot 2023-11-06 at 20 42 04](https://github.com/WDaan/VueTorrent/assets/428231/6dec672c-1aa8-445c-9ef9-c1845d5dfe18)

I downloaded my version using the command: `git clone --single-branch --branch latest-release https://github.com/WDaan/VueTorrent.git` and if I run `git pull` it tells me it is already updated.
But the last stable release in [Releases page](https://github.com/WDaan/VueTorrent/releases) is 2.0.1 😕 

Having clarified this, the bug I want to report is that when I modify something in the `Torrent card` section of the settings and press save, it shows the torrents with those settings correctly but if I refresh the page it reverts to the default values.

**Comment by WDaan** [comment]
I thought this bug was resolved. Also there seems to have been an issue with the versioning 😬
We'll get that sorted out with the next release

**Comment by Larsluph** [comment]
This has already been fixed but in the 2.1.0 version which had a problem and didn't release completely.

**Comment by simonbcn** [comment]
I don't understand why you close this issue without having released the version that fixes the problem.
